setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='quickscreen',
+    version='0.1',
+    packages=find_packages(exclude=['tests*']),
+    license='MIT',
+    description='A package for exploring data',
+    url='https://github.com/mqharris/533_lab2',
+    author='Nathan Smith, Mitch Harris, Ryan Koenig',
+    author_email='blank'
```

Comment:
Left this blank
But I might need to put an email in that is registered to PyPi

Reply:
how is the description.md, demo.ipynb, and data folder getting screened out?